backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -179,7 +182,8 @@ func (fs *experimentStore) GetExperiments(ctx context.Context, configType string
 
 		anyConfig := &any.Any{}
 		if nil != jsonpb.Unmarshal(strings.NewReader(details), anyConfig) {
-			return nil, err
+			fs.getExperimentsLoadConfigurationError.Inc(1)
```

Comment:
I'd log the error here to make it easier to find out why experiments can't be unmarshalled.

I am not a big fan of skipping errors, because they can hide real problems. But in this case skipping some experiments could provide a better UX than failing if there are multiple experimentation modules and one is missing.

So I think skipping here would be acceptable if we recommend users to set an alarm on this metric.

Reply:
addressed